packages/react-ui/webpack.config.js

```diff
@@ -8,6 +8,7 @@ const webpackConfig = {
   ...webpackBaseConfig,
   externals: [ 
     /^@material-ui\/.+$/,
+    '@carto/react-core',
```

Comment:
In react-api we use something like: 
```
  externals: [
    /^@carto\/react-.+$/,
```

Reply:
Yes, but I guessed that we don't need every package.